make/modules/jdk.incubator.vector/Lib.gmk

```diff
@@ -37,3 +37,16 @@ ifeq ($(call isTargetOs, linux windows)+$(call isTargetCpu, x86_64)+$(INCLUDE_CO
 
   TARGETS += $(BUILD_LIBJSVML)
 endif
+
+ifeq ($(call isTargetOs, linux)+$(call isTargetCpu, aarch64)+$(INCLUDE_COMPILER2), true+true+true)
+  $(eval $(call SetupJdkLibrary, BUILD_LIBVECTORMATH, \
+      NAME := vectormath, \
+      CFLAGS := $(CFLAGS_JDKLIB) -Wno-error=unused-function, \
```

Comment:
Should the unused-function be passed in using `DISABLE_WARNINGS_*` instead?

Reply:
Thanks!
Good suggestion, it makes the output clean.